cmd/changelog-parser/main.go

```diff
@@ -247,6 +253,8 @@ func main() {
 		"Output filename")
 	flag.StringVar(&version, "v", defaultVersionString,
 		"Version to embed in the changelog")
+	flag.StringVar(&apiToken, "p", "",
```

Comment:
Should we document these command line flags in the README.md?

Reply:
Yeah - good point. I'll do that once we figure out what to name this flag.

Reply:
"p" is good for me!